data/tools/dataflow-framework.yml

```diff
@@ -0,0 +1,11 @@
+name: Dataflow Framework
+categories:
+  - linter
+tags:
+  - java
+license: GPL with Classpath exception / MIT License
+types:
+  - cli
+source: 'https://github.com/typetools/checker-framework'
+homepage: 'https://checkerframework.org/manual/checker-framework-dataflow-manual.pdf'
```

Comment:
Not sure if we should use a pdf as the homepage. We could use the same url as for source and add the pdf as a resource maybe?